tests/conftest.py

```diff
@@ -15,6 +16,14 @@
 os.environ[cast_bytes_py2("DVC_TEST")] = cast_bytes_py2("true")
 
 
+# Make DVC tests use separate OAuth token to access Google Drive
+def skip_pydrive_init(_):
+    pass
+
+
+RemoteGDrive.init_gdrive = skip_pydrive_init
```

Comment:
Feels weird to patch it like this in conftest. Maybe mock.patch where needed would be better?

Reply:
@MaxRis What do you think? ^

Reply:
Though, if we don't provide default credentials, then we won't need this hack too, right?

Reply:
It's left over, actually (already used `mock.patch` in corresponding places), and I've thought I've removed this hack already, but for some reasons have missed it. Sorry about that.
Will remove this.

Currently, there are 2 ways to configure different parts of authentication process:
1. Config.SECTION_GDRIVE_CREDENTIALPATH  path to custom PyDrive's settings file (it configures where and how auth2 tokens are saved and what disk permissions DVC gets).
2. `PYDRIVE_USER_CREDENTIALS_DATA` env variable with sensitive auth2 token data to be used for google authorization (usage is supposed for automated tests).

> Though, if we don't provide default credentials, then we won't need this hack too, right?

Yes, it's already not needed to have this in the place.